zubhub_backend/zubhub/creators/views.py

```diff
@@ -79,10 +79,9 @@ def get(self, request, format=None):
 
 class UserProfileAPIView(RetrieveAPIView):
     """
-    Fetch Profile of user with given username.
-
-    Requires username of user.
-    Returns user profile.
+    Note that this schema returns the full user profile, but the api sometimes
```

Comment:
The docstring should be added to the previous docstring, not replace it. It should look more like 
    
"""
Fetch Profile of user with given username.
Requires username of user.
Returns user profile.

Note that this schema returns the full user profile, but the api sometimes
returns a minimal version of the user profile, omitting certain fields that
are not neccessary or are sensitive.
"""

Reply:
@NdibeRaymond, I have fixed it. Now could you please check?